### PR TITLE
Restructuring of repo

### PR DESCRIPTION
**Pull request by jenniferthompson**
I think I have it cleaned up - anything y'all see that I missed or should be different? I made folder names generally consistent. I have empty folders created on my drive for some things that aren't showing up here (R/d4ddrugspending, eg) because there aren't any files in them yet.

### CODE REVIEW

**Comment by jenniferthompson** [comment]
Not sure if it would be better to go ahead and merge @cduvallet's PR first, then incorporate that here, or the other way around?

**Comment by dhuppenkothen** [comment]
For the empty folders, you could make a bare-bones `README.md` in each that will work as a sort of directory for the repo structure? Committing that, the folders will appear.

File: .gitignore
Comment (dhuppenkothen) [inline .gitignore]: Did you delete the `.gitignore` by accident or is there a reason behind it?

**Review by dhuppenkothen — COMMENTED** [review]
It's a bit hard to see what the re-organization really looks like. Based on what I can see, I think all the `centerwatch` stuff should be in `d4ddrugwrangling` (which would become the Python package), not `datawrangling`, since it defines importable classes, has an `__init__.py` file, etc (all things I'd expect for a Python package). 

Unless @mattgawarecki disagrees?

**Comment by dhuppenkothen** [comment]
Let's get the directory structure sorted first. Once that's done, I may be able to write a couple of simple Python scripts that allow one to import code, which would I think help @cduvallet with not duplicating code that's already there.

File: .gitignore
Comment (mattgawarecki) [reply .gitignore]: We definitely wanna keep the .gitignore in one form or another.

Comment (jenniferthompson) [reply .gitignore]: It had been added by one person as part of a PR; it wasn't a broad group one. Could add it back and add more file types if needed.

**Comment by jenniferthompson** [comment]
Ah, yep, good call on READMEs; we'll definitely want those anyway. I'll try to do that today sometime. I defer to you two on Python organization!

File: .gitignore
Comment (jenniferthompson) [reply .gitignore]: Typing at the same time @mattgawarecki! I'll add it back and see if there are any file types to be added.

**Comment by mattgawarecki** [comment]
Review comments:

1. I think we can safely get rid of `cms/`, since that data's up in data.world. (Probably just a result of accidentally removing `.gitignore`.)
2. I've seen in other projects where all-caps filenames are used for basically all documentation (licensing, readmes, etc.). Maybe this applies to `datadictionaries/data_dictionary_template.md`? Thinking maybe just `datadictionaries/TEMPLATE.md`.
3. Probably safe to remove the suffix on `datadictionaries/manufacturers_datadict.md` (i.e., just `datadictionaries/manufacturers.md`)
4. Re: `python/datawrangling/centerwatch` -- I'm fine with it going in `python/d4ddrugspending` since it's sort of a sub-module. Just for clarification, it's a Scrapy spider that gathers data from a site called "Centerwatch."
5. If we do the above, we can safely get rid of `python/datawrangling`.
6. If you don't foresee us writing READMEs for directories that still need to exist, you can simply create an empty file (conventionally, `.gitkeep`) and Git will track that file, which guarantees the directory will exist when people clone the repo.
7. The rest of the structure looks good to me, I think.

**Comment by cduvallet** [comment]
I think it makes more sense to merge this PR first. I can then move around my files to be in the right place, import code from @dhuppenkothen's scripts, etc. This way I can also be the first guinea pig for the new structure ;)

**Comment by jenniferthompson** [comment]
@mattgawarecki @dhuppenkothen Made the changes - let me know if I missed anything! I left `python/datawrangling` for now but feel free to remove it if it's never going to be useful.

ETA - going to work on edits to the main README to reflect these changes later this afternoon/evening, so it's not 100% ready to go yet, but that's the only change I have planned unless you two see anything else.

**Comment by jenniferthompson** [comment]
README is edited! Ready to merge when y'all are.

**Review by mattgawarecki — APPROVED** [review]

**Pull request merged**
